Meter.c

```diff
@@ -308,10 +308,8 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
       memset(data->values, 0, (data->nValues - oldNValues) * sizeof(*data->values));
    }
    const size_t nValues = data->nValues;
-   if ((size_t)w * 2 > nValues) {
-      x += w - nValues / 2;
-      w = nValues / 2;
-   }
+   if (nValues <= 0)
+      return;
```

Comment:
`size_t nValues` cannot be less than 0; only equal … I'd prefer `nValues < 1` for clarity instead.